clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -524,7 +522,7 @@ public boolean coordinatorUnknown() {
      */
     protected void coordinatorDead() {
         if (this.coordinator != null) {
-            log.info("Marking the coordinator {} dead.", this.coordinator.id());
+            log.info("Marking the coordinator {} dead. Find the new coordinator and retry.", this.coordinator.id());
```

Comment:
Should the second sentence be "Will find new coordinator and retry"? That seems to be what we use everywhere else.